docs/source/using/managing-minishift.adoc

```diff
@@ -89,6 +89,11 @@ $ minishift start --iso-url https://github.com/minishift/minishift-centos-iso/re
 ----
 $ minishift start --iso-url file:///<path_to_ISO_image>
 ----
```

Comment:
Can you also fix that the previous example i.e. `$ minishift start --iso-url file:///<path_to_ISO_image>` and mention it is for macOS and GNU/Linux.

Reply:
Correction: `file://` (not three, as the third is part of the path already). Our documentation does not mention this correctly

Reply:
It is better to mention it is an absolute path...

Reply:
`file:///` is shown as an example, and so is the case for the drive letter as a windows example. `file://c:/`.
 we can mention it as an absolute path and show only one example.